mods/cnc/maps/twist-of-fate/rules.yaml

```
@@ -0,0 +1,113 @@
+World:
+	MissionData:
+		StartVideo: airstrk.vqa
+		LossVideo: obel.vqa
+		WinVideo: paratrop.vqa
+		Briefing: We had set up a small recon post in Nod territory, but they captured and reinforced it.\n\nGetting that post back would be a major coup.\n\nAn MCV and armed convoy are on their way to aid you in establishing a new base.\n\nOnce established, eliminate all Nod forces in the area.
+	LuaScript:
+		Scripts: campaign.lua, utils.lua, twist-of-fate.lua, twist-of-fate-AI.lua
+	ScriptLobbyDropdown@difficulty:
+		ID: difficulty
+		Label: dropdown-difficulty.label
+		Description: dropdown-difficulty.description
+		Values:
+			easy: options-difficulty.easy
+			normal: options-difficulty.normal
+			hard: options-difficulty.hard
+		Default: normal
+
+Player:
+	PlayerResources:
+		DefaultCash: 0
+
+CAMERA.mini:
+	Inherits: CAMERA
+	RevealsShroud:
+		Range: 1c512
+		Type: CenterPosition
+	RenderSpritesEditorOnly:
+		Image: camera
+
+FLARE:
+	RevealsShroud:
+		Range: 7c0
+
+A10:
+	Targetable:
```

This needs to be
```suggestion
	Targetable:
		TargetTypes: Air
```
for it to work. The planes are way too faster for all anti-air though.